scripts/actions/send-and-update-translation-queue.js

```diff
@@ -77,13 +77,60 @@ const uploadFile = (locale, batchUid, accessToken) => async (page) => {
 
   if (code === 'ACCEPTED' && resp.ok) {
     console.log(`[*] Successfully uploaded ${page.file}.`);
+    await sendPageContext(page.file, accessToken);
   } else {
     console.error(`[!] Unable to upload ${page.file}.`);
   }
 
   return { code, locale, slug: page.file };
 };
 
+/**
+ * sends the html file as a visual context for each uploaded file
+ * @param {string} fileUri
+ * @param {string} accessToken
+ * @returns {Promise}
+ */
+const sendPageContext = async (fileUri, accessToken) => {
+  const filepath = fileUri.replace(`src/content/`, '');
+  const slug = filepath.replace(`.mdx`, '');
+  const contextUrl = new URL(slug, 'https://docs-preview.newrelic.com'); //need to change this once we migrate to docs-newrelic-com
```

Comment:
Perhaps w should move this into a constant at the opt of the file like we did with `LOCALE_IDS` and `PROJECT_ID`

Reply:
I like that idea! that's what I was also thinking, because then we just have to change the var instead of finding where it is in this file :)